spec/routing/user_event_preferences_routing_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe User::UserEventPreferencesController, type: :routing do
+  describe "routing" do
+    it "routes to #index" do
+      expect(get: "/user_event_preferences").to route_to("user_event_preferences#index")
```

Comment:
something smells bad here.. I expect a route like `user/1/user_event_preferences` to route to `user_event_preferences#index` otherwise you will see all the preferences for all the users altogether...